### PR TITLE
Knitr vignettes

### PR DESCRIPTION
**Pull request by eoduniyi**

### CODE REVIEW

File: project.Rproj [hunk 1]
```diff
@@ -18,5 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-PackageCheckArgs: --as-cran
```
Comment (GegznaV) [inline project.Rproj:21]: Why is this line removed?

Comment (eoduniyi) [reply project.Rproj:21]: > Why is this line removed?

I'm not sure. My build settings seem right, and when the package is checked it passes all the R CMD checks with 1 NOTE:

Screen Shot 2021-02-23 at 8 48 14 AM

```
> checking for portable file names ... NOTE
  Found the following non-portable file path:
    hySpc.chondro/vignettes/hySpc-chondro_cache/html/chondro-import_159d9d14e642eb48cb4935f7e3a62bd5.RData
  
  Tarballs are only required to store paths of up to 100 bytes and cannot
  store those of more than 256 bytes, with restrictions including to 100
  bytes for the final component.
  See section ‘Package structure’ in the ‘Writing R Extensions’ manual.

0 errors ✓ | 0 warnings ✓ | 1 note x
```
